docs/lib/datastore/fragments/ios/datastore-events.md

```diff
@@ -0,0 +1,50 @@
+## Usage
+For instance, to see if the network status is active, you could set up the following listener or Subscriber (using Combine):
```

Comment:
I think a more introductory paragraph is missing. Introducing what hub events are and what use cases they cover.

Also, this content can/should be moved to the `docs/lib/datastore/datastore-events.md` file since it's common to all platforms.